mcc/odin/odin/snapshot_list.go

```diff
@@ -0,0 +1,18 @@
+package odin
+
+import (
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/aws/aws-sdk-go/service/rds/rdsiface"
+)
+
+// ListSnapshots return a list of all DBSnapshots.
+func ListSnapshots(
+	instanceName string,
+	svc rdsiface.RDSAPI,
```

Comment:
Yea, that's about as much as I figured 😄 

My question should have been: What does `svc` stand for (and can we maybe change it to a bit more verbose name) and what type of data does it hold? :)